src/account_info.py

```diff
@@ -5,40 +5,69 @@
 class AccountInfo:
     def __init__(self):
         try:
-            self.database = 'data/people_data-copy.csv'
+            self.database = 'data/UserInfo.csv'
             self.data = pd.read_csv(self.database)
         except FileNotFoundError:
-            self.database = '../data/people_data-copy.csv'
+            self.database = '../data/UserInfo.csv'
             self.data = pd.read_csv(self.database)
         
-    def create_account(self, name, surname='', birthday='', interests='', wishlist='', friendlist=''):
-        id_list = sorted(self.data.ID.tolist(), reverse=True)
-        lastID = id_list[0]
-        if name == '':
-            print('Name cannot be empty')
-            return -1
-        else:
+    def create_account(self, name, surname='', birthday='',username='', password = '',interests='', wishlist='', friendlist=''):
+        print("whole data")
+        if self.data.empty:
+            if name == '':
+                print('Name cannot be empty')
+                return -1
+            
             account_dict = {
-                'ID': lastID+1,
-                'Name': name,
-                'Surname': surname,
+                'ID': 0,
+                'First Name': name,
+                'Last Name': surname,
                 'Birthday': birthday,
+                'UserName' : username,
+                'Password' : password,
                 'Interests': interests,
                 'WishList': wishlist,
                 'FriendList': friendlist
             }
             self.data = self.data.append(account_dict, ignore_index=True)
             self.data.to_csv(self.database, index=False)
             print('Account created successfully!')
-        return self.data[self.data['ID']==lastID+1]
+            return self.data[self.data['ID']==1]
+        else:
+            id_list = sorted(self.data.ID.tolist(), reverse=True)
+            lastID = id_list[0]
+            if name == '':
+                print('Name cannot be empty')
+                return -1
+            elif username in self.data['UserName'].values:
+                print('User Name already in use. Please use another one!')
+                st.error("User Name already in use. Please use another one!")
```

Comment:
This looks like an abbreviation of streamlit and it's causing a compilation error. I think we need to add `import streamlit as st` to this file